src/main/kotlin/com/criptext/mail/scenes/composer/ComposerController.kt

```diff
@@ -50,7 +50,12 @@ class ComposerController(private val storage: KeyValueStorage,
     private val dataSourceController = DataSourceController(dataSource)
     private val observer = object: ComposerUIObserver {
         override fun onSenderSelectedItem(sender: String) {
-            model.selectedAccount = model.accounts.find { it.userEmail == sender }
+            val lastIndexParenthesis = sender.lastIndexOf("(")
```

Comment:
It doesn't look good. What if `sender` is an object or the info needed already comes in the `onSenderSelectedItem` function.